src/mmw/js/src/core/layerPicker.js

```diff
@@ -312,63 +241,11 @@ var LayerPickerView = Marionette.LayoutView.extend({
     },
 
     collectionEvents: {
-        'change': 'render'
+        'change': 'render',
+        'change:layer': 'render'
```

Comment:
`render` should only trigger once because `change:layer` is actually a (poorly named) custom event. Because the layers themselves are nested so deeply in the `LayerTabCollection`, toggling a single layer doesn't fire a regular `change` event on the collection, we have to trigger it "manually".

I'll rename it to something like `toggle:layer`